app/views/movies/show.html.erb

```diff
@@ -0,0 +1,16 @@
+<h2><%= @movie.title %></h2>
+
+<%= "Year of Creation: #{@movie.creation_year}" %> <br>
+<%= "Genre: #{@movie.genre}" %> <br>
+  <br>
+<h4>Actors, Youngest to Oldest:</h4>
+<% @movie.actors_youngest_to_oldest.each do |actor| %>
+  <%= actor %>
+<% end %>
+<br><br>
+<%= "Average age of actors: #{@movie.average_age_of_actors}" %>
+<br><br>
+<%= form_with url: "/movies/#{@movie.id}", method: :patch, local: :true do |f| %>
```

Comment:
The form looks good, but I think instead of `:patch` here we should use `:post`, since we are _creating a relationship_ between this movie and the actor by the ID we provide. `PATCH` and `PUT` are verbs we should use when updating a resource - and if we are updating this movie resource, we would change one of the  values of the `movies` table. Since that's not what we're doing, a different verb should be used, in this case POST.